task/s2i-angular-nginx/0.1/s2i-angular-nginx.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  annotations:
+    tekton.dev/displayName: s2i angular nginx
+    tekton.dev/categories: Image Build
+    tekton.dev/tags: image, build, angular, nginx
+    tekton.dev/pipelines.minVersion: '0.19'
+  name: s2i-angular-nginx
+  labels:
+    app.kubernetes.io/version: '0.1'
+spec:
+  params:
+    - default: '14'
```

Comment:
The latest active NodeJS LTS version is 16. Let's use that as the default.

Angular requires an [active LTS or maintenance LTS](https://nodejs.org/about/releases) version of Node.js.
https://angular.io/guide/setup-local#prerequisites